webapp/channels/src/components/admin_console/access_control/table_editor/table_editor.tsx

```diff
@@ -0,0 +1,254 @@
+                                                    <button
+                                                        key={value}
+                                                        className='table-editor__dropdown-value'
+                                                        onClick={() => addCondition(attr.attribute, value)}
+                                                    >
+                                                        {value}
+                                                    </button>
+                                                ))}
+                                            </div>
+                                        </div>
+                                    ))}
+                                </div>
+                            </div>
+                        ) : (
+                            <button
+                                className='table-editor__add-button'
+                                onClick={() => setShowAttributeDropdown(true)}
+                            >
+                                <i className='icon icon-plus'/>
+                                <FormattedMessage
+                                    id='admin.access_control.table_editor.add_condition'
+                                    defaultMessage='Add condition'
+                                />
+                            </button>
+                        )}
+                    </div>
+                )}
+            </div>
+            {serverError && (
+                <span className='EditPolicy__error'>
+                    <i className='icon icon-alert-outline'/>
+                    <FormattedMessage
+                        id='admin.access_control.edit_policy.serverError'
+                        defaultMessage='There are errors in the form above'
+                    />
+                </span>
+            )}
+            <div className='table-editor__footer'>
+                <div className='table-editor__footer-right'>
+                    <button
+                        className='table-editor__button table-editor__button--test'
+                        onClick={testAccessRule}
+                        disabled={disabled || !value}
+                    >
+                        <i className='icon icon-lock-outline'/>
+                        <FormattedMessage
+                            id='admin.access_control.table_editor.test_access_rule'
+                            defaultMessage='Test access rule'
+                        />
+                    </button>
+                </div>
+            </div>
+            <div className='table-editor__help-text'>
+                <Markdown
+                    message={'Add rules to combine user attributes to restrict channel membership. This is a simple expression editor and does not support complex expressions.'}
+                    options={{mentionHighlight: false}}
+                />
+                <a
+                    href='#'
```

Comment:
I believe this link needs to be updated when we have the corresponding docs ready? If so, can you please add a comment here (and create a jira ticket if you want) to track it?